lineman/app/app.coffee

```diff
@@ -15,6 +15,9 @@ angular.module('loomioApp', ['ngNewRouter',
   renderer = new marked.Renderer()
   renderer.link = (href, title, text) ->
     "<a href='" + href + "' title='" + (title || text) + "' target='_blank'>" + text + "</a>";
+  renderer.paragraph = (text) ->
+    text = text.replace(/([\s(\[]+|^)@([a-zA-Z0-9]+)([\s)\]\,\?]|$)/g, "$1<a class='lmo-user-mention' href='/u/$2'>@$2</a>$3")
```

Comment:
Hm, additionally, it seems that the twitter urls (http://www.twitter.com, etc.) are hard-coded into the gem; we could change them, but we'd have to overwrite a frozen constant, or maybe fork the repo. Starting to wonder if I can find whatever regex they're using and steal it instead.